dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -689,7 +689,9 @@ export const LiveLayout = ({ CAPI, NAV, format, palette }: Props) => {
 								</GridItem>
 								<GridItem area="meta" element="aside">
 									<Hide until="desktop">
-										<div css={maxWidth}>
+										<div
+											css={[maxWidth, sidePaddingDesktop]}
```

Comment:
What does this change do?

Reply:
Ideally, css relating to the ArticleMeta component would exist inside the component. We have these `maxWidth` divs as a sort of exception but like all exceptions they invite expansion!

Reply:
Ah yes. That change accidentally crept in to the branch. Small one to edit the positioning on the meta
<img width="242" alt="Screen Shot 2021-12-17 at 17 41 10" src="https://user-images.githubusercontent.com/2051501/146585989-616e12d9-ecf1-4b6a-b519-9ff202a1de9c.png">
<img width="242" alt="Screen Shot 2021-12-17 at 17 41 22" src="https://user-images.githubusercontent.com/2051501/146585995-8100ef83-f414-4cf2-bf03-ce4d22231cca.png">

Reply:
@oliverlloyd 🆙

Reply:
Hmm, it seem there's already quite a lot of padding and margin css in this layout file now. I guess this could be reviewed at a later stage but now isn't the time